index.md

```diff
@@ -154,11 +155,11 @@ Answered questions from the issue tracker will be transferred to the [FAQ](https
 - One team can only get one prize.
 - One Person can only join on team.
 - Model training is not allowed use ground truth/observations data after forecast was issued, see Data Timings.
-- All taxes imposed on prizes are the sole responsibility of the winners.
-- By joining the competition (see steps https://renkulab.io/projects/aaron.spring/s2s-ai-challenge-template), participants agree that they will make their private repositories on renkulab.io public after the competition ends (31st October 2021) regardless whether their contributions are among the top 3 for prizes. All repositories must be made availbale under the xyz-tbd licence.
-- warranty?
-- law?
-- some WMO liability statement?
+- All taxes imposed on prizes are the sole responsibility of the winners. 
```

Comment:
taken from kaggle: https://www.kaggle.com/c/coleridgeinitiative-show-us-the-data/rules